woocommerce/facebook-for-woocommerce
# Update composer dependencies to fix broken repository

### Changes proposed in this Pull Request:
Running `composer install` is currently broken on the `master` branch. This PR updates the composer dependencies to pull in the latest working dependencies.

### How to test the changes in this Pull Request:
1. Run `composer install` on the `master` branch to see that there is an error
2. Run `composer install` on this branch to see that the installation completes as expected

### Changelog entry


> Dev - Update composer dependencies
